jre_emul/android/platform/libcore/harmony-tests/src/test/java/org/apache/harmony/tests/java/util/zip/DeflaterInputStreamTest.java

```diff
@@ -324,19 +344,20 @@ public void testSkip() throws IOException {
         } catch (IOException e) {
             // expected
         }
-
-        is = new ByteArrayInputStream(TEST_STR.getBytes("UTF-8"));
-        dis = new DeflaterInputStream(is);
-        assertEquals(23, dis.skip(Long.MAX_VALUE));
-        assertEquals(0, dis.available());
     }
 
     /**
      * DeflaterInputStream#DeflaterInputStream(InputStream)
      */
-    public void testDeflaterInputStreamInputStream() {
+//    @DisableResourceLeakageDetection(
```

Comment:
Again, follow the J2ObjC convention for commenting lines out.

Reply:
added J2ObjC comments